aldryn_faq/boilerplates/bootstrap3/templates/aldryn_faq/category_list.html

```diff
@@ -2,11 +2,14 @@
 {% load i18n cms_tags %}
 
 {% block faq_sidebar %}
-    {% static_placeholder "faq_sidebar_top" %}
+    {% render_placeholder view.config.placeholder_faq_sidebar_top %}
+    {% static_placeholder "faq_sidebar_top" %}{# DEPRECATED #}
```

Comment:
I'll still do it in a subsequent release because it provides a migration path for existing projects that use FAQ.